lib/utils/usage_docs.go

```diff
@@ -0,0 +1,299 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//go:build docs
+
+package utils
+
+import (
+	"bytes"
+	_ "embed"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/alecthomas/kingpin/v2"
+)
+
+// formatThreeColMarkdownTable formats the provided row data into a three-column
+// Markdown table, minus the header.
+func formatThreeColMarkdownTable(rows [][3]string) string {
+	buf := bytes.NewBuffer(nil)
+	for _, r := range rows {
+		buf.WriteString("\n|" + r[0] + "|" + r[1] + "|" + r[2] + "|")
```

Comment:
This could also be expressed as `fmt.Fprintf(buf, "\n|%v|%v|%v|", r[0], r[1], r[2])` if you like that better. No strong preference on my side.